hw3.ipynb

```diff
@@ -0,0 +1,895 @@
+{
```

Comment:
No absolute paths

Reply:
Consider https://github.com/pykaldi/pykaldi to make it more readable

Reply:
No one really does fully-connected autoencoders. Especially this deep

Reply:
@GostD I can see you struggles with computations: why did you not use a server or something?